tools/scenario-player/scenario_player/tasks/services.py

```diff
@@ -0,0 +1,122 @@
+from scenario_player.exceptions import ScenarioAssertionError, ScenarioError
+from scenario_player.tasks.api_base import RESTAPIActionTask
+
+
+class PFSAssertTask(RESTAPIActionTask):
```

Comment:
This is nice, but we'll need some time to implement the debug API. Can we add an assertion like `assert_pfs_routes: {source: 0, target: 1, num_routes: 2}` to start and be able to check the basic topology and network infos the PFS creates?

This doesn't need to be done in this PR and I can do/help with the implementation.